LogicalTypes.md

```diff
@@ -29,8 +29,215 @@ This file contains the specification for all logical types.
 
 ### Metadata
 
-The parquet format's `ConvertedType` stores the type annotation. The annotation
+The parquet format's `LogicalType` stores the type annotation. The annotation
```

Comment:
Two sentence after it is mentioned, that in fact LogicalType have an older representation called ConvertedType. Isn't it enough to mention it there?

Reply:
Yes, ConvertedType is mentioned but the actual types like INT32, INT64 etc. are removed and only the new LogicalTypes are listed. I think, it would be better to have all the deprecated ConvertedTypes described as well.
As you already have different sections for the different types like integer, I would suggest describing both types in the related section. We might highlight it with a different format or under a sub-header so it will be obvious that these types are deprecated.

Reply:
Ah okay got it. Makes sense.